app/tests/pages/index.test.tsx

```diff
@@ -39,4 +58,26 @@ describe('The index page', () => {
 
     expect(screen.getByText('Go to dashboard')).toBeInTheDocument();
   });
+
+  it('Does not display alert message when there is an open intake', async () => {
+    pageTestingHelper.loadQuery();
+    pageTestingHelper.renderPage();
+
+    expect(
+      screen.queryByText(
+        `New applications will be accepted after updates to ISED‘s Eligibility Mapping tool are released.`
```

Comment:
Same comment as the previous PR regarding having this text in a variable. If we're asked to change the text, this test will not fail (but it should)

Reply:
Sounds good, moved the message to a constant.